src/drive/web/modules/viewer/Panel/getPanelBlocks.jsx

```diff
@@ -1,8 +1,16 @@
-import { hasCertifications } from 'drive/web/modules/viewer/helpers'
+import KonnectorBlock from 'cozy-harvest-lib/dist/components/KonnectorBlock'
+
+import {
+  hasCertifications,
+  isFromConnector
+} from 'drive/web/modules/viewer/helpers'
 import Certifications from './Certifications'
 
-// TODO add connector block
 export const panelBlocksSpecs = {
+  connector: {
```

Comment:
let's keep Konnector, no? Sometimes we have Connector in your codebase sometimes Konnector. We should stay on Konnector?